tests/integration/test.uuids.js

```diff
@@ -37,32 +37,9 @@ describe('test.uuid.js', function () {
       .length(count, 'Generated small UUIDS are unique.');
   });
 
-  it('Test custom length', function () {
-    var length = 32;
-    var count = 10;
-    var uuids = makeUuids(count, length);
-    // Test single UUID wrapper
-    uuids.push(testUtils.uuid(length));
-    uuids.map(function (uuid) {
-      uuid.should.have.length(length, 'UUID length is correct.');
-    });
-  });
+  it('_rev generation', function () {
```

Comment:
As a future note as this was broken before you touched it, but integration tests are for public api stuff in general, this could just be a unit test if you think its even needed at all